e2e/Utils.ts

```diff
@@ -260,3 +261,12 @@ export const normalizeIcons = (str: string) => {
     .replace(new RegExp('\u00D7', 'g'), '\u2715')
     .replace(new RegExp('\u221A', 'g'), '\u2713');
 };
+
+// Certain environments (like CITGM and GH Actions) do not come with mercurial installed
+export const buildTestIfHg = () => {
+  const hgIsInstalled = which.sync('hg', {nothrow: true}) !== null;
+  if (!hgIsInstalled) {
+    console.warn('Mercurial (hg) is not installed - skipping some tests');
+  }
+  return hgIsInstalled ? test : test.skip;
+};
```

Comment:
Alternatively we might cache the `hgIsInstalled`. But I am not sure it will be required now as only specs with tests relying on mercurial would need to call it.

One possible advantage of this function is that it could be adapted to check for other programs than just hg.